esrally/metrics.py

```diff
@@ -1069,6 +1095,18 @@ def _get(self, name, task, operation_type, sample_type, node_name, mapper):
                 (node_name is None or doc.get("meta", {}).get("node_name") == node_name)
                 ]
 
+    def get_one(self, name, sample_type=None, node_name=None, task=None, mapper=lambda doc: doc["value"],
+                sort_key=None, sort_reverse=False):
+        if sort_key:
+            docs = sorted(self.docs, key=lambda k: k[sort_key], reverse=sort_reverse)
+        else:
+            docs = self.docs
+        for doc in docs:
```

Comment:
I think we need to `return None` if there are no hits (i.e. after the for loop).